src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-list/rbd-list.component.ts

```diff
@@ -214,14 +214,18 @@ export class RbdListComponent extends ListWithDetails implements OnInit {
       icon: Icons.edit,
       click: () => this.actionPrimary(true),
       name: this.actionLabels.PROMOTE,
-      visible: () => this.selection.first() != null && !this.selection.first().primary
+      visible: () => this.selection.first() != null && !this.selection.first().primary,
+      disable: () =>
+        this.selection.first().mirror_mode === 'Disabled' ? 'Mirroring must be enabled' : ''
```

Comment:
```suggestion
        this.selection.first().mirror_mode === 'Disabled' ? 'Mirroring needs to be enabled on the image to perform this action' : ''
```

Reply:
updated it

Reply:
i don't see the change there? @Pegonzal

Reply:
I mistakenly pushed the wrong changes, thanks for pointing it out @nizamial09, it should be good to go now

Reply:
thank you!